www/src/components/CodeBlock.tsx

```diff
@@ -32,12 +32,30 @@ export type CollapsibleLiveEditorTypes = {
 
 function CollapsibleLiveEditor({ children }: CollapsibleLiveEditorTypes) {
   const [collapseIsOpen, setCollapseIsOpen] = useState(false);
+
+  const eventSegmentSubmitter = (e: Event & { target: HTMLElement }) => {
```

Comment:
nit: function names should usually be verbs, maybe rename it to something like `submitSegmentEvent`?

Reply:
Done